src/common/reader.js

```diff
@@ -1197,8 +1199,37 @@ class Reader {
 						}
 						else {
 							this._lastView.navigate({ annotationID: annotation.id });
+							// When annotation is added from "Find in" popup, close it and re-focus the view
+							this.toggleFindPopup({ primary: this._lastViewPrimary, open: false });
```

Comment:
This can probably go into find popup component as it already has a way to close itself.

Reply:
Gotcha, I'll keep it in mind! I re-pushed everything as it was easier to find the double quotes that way